frontends/concrete-python/examples/prime-match/prime-match-sh.py

```diff
@@ -0,0 +1,113 @@
+import random
+import time
+
+import numpy as np
+
+from concrete import fhe
+
+NUMBER_OF_SYMBOLS = 10
+
+MINIMUM_ORDER_QUANTITY = 1
+MAXIMUM_ORDER_QUANTITY = 50
+
+assert 0 < MINIMUM_ORDER_QUANTITY < MAXIMUM_ORDER_QUANTITY
+
+
+def prime_match(
+    bank_order_quantities,
+    client_order_quantities,
+):
+    with fhe.tag("calculating-matching-order-quantity"):
+        return np.minimum(bank_order_quantities, client_order_quantities)
+
+
+inputset = [
+    (
+        np.random.randint(
+            MINIMUM_ORDER_QUANTITY, MAXIMUM_ORDER_QUANTITY, size=(NUMBER_OF_SYMBOLS * 2,)
+        ),
+        np.random.randint(
+            MINIMUM_ORDER_QUANTITY, MAXIMUM_ORDER_QUANTITY, size=(NUMBER_OF_SYMBOLS * 2,)
+        ),
+    )
+    for _ in range(1000)
+]
+configuration = fhe.Configuration(
+    enable_unsafe_features=True,
+    use_insecure_key_cache=True,
+    insecure_key_cache_location=".keys",
+    fhe_simulation=True,
```

Comment:
`fhe_simulation=True`: do we really execute in FHE? I would say Yes since timings are long but checking. CC @umut-sahin

Reply:
Yes, `fhe_simulation` is just enabling simulation, it's not making `encrypt/run/decrypt` simulated.

Reply:
ok thx